internal/tofu/context_plan.go

```diff
@@ -568,32 +563,100 @@ func (c *Context) postPlanValidateImports(importResolver *ImportResolver, allIns
 func (c *Context) findImportTargets(config *configs.Config, priorState *states.State) []*ImportTarget {
 	var importTargets []*ImportTarget
 	for _, ic := range config.Module.Import {
-		if priorState.ResourceInstance(ic.To) == nil {
-			importTargets = append(importTargets, &ImportTarget{
-				Config: ic,
-			})
-		}
+		importTargets = append(importTargets, &ImportTarget{
+			Config: ic,
+		})
 	}
 	return importTargets
 }
 
-func (c *Context) validateImportTargets(config *configs.Config, importTargets []*ImportTarget) (diags tfdiags.Diagnostics) {
+// validateImportTargets makes sure all import targets are not breaking the following rules:
+//  1. Imports are attempted to resources that do not exist (if config generation is not enabled).
```

Comment:
tiny tiny tiny nitpick
```suggestion
//  1. Imports are attempted in to resources that do not exist (if config generation is not enabled).
```

Reply:
https://github.com/opentofu/opentofu/pull/1270/commits/a02afd9341273a015d3f1c4eba0ecbee0ddbeb41